### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**
Based on https://github.com/mattermost/desktop/blob/release-5.0/package.json#L124

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -449,6 +591,19 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
...
+## sass
+
+This product contains 'sass' by sass.
+
+Sass makes CSS fun again. Sass is an extension of CSS, adding nested rules, variables, mixins, selector inheritance, and more. It's translated to well-formatted, standard CSS using the command line tool or a plugin for your build system.
+
+* HOMEPAGE:
+  * https://github.com/sass/sass
+
+* LICENSE: ?
```
Comment (amyblais) [inline NOTICE.txt:603]: No license?

Comment (devinbinnie) [reply NOTICE.txt:603]: https://sass-lang.com/
MIT License at the bottom of the page.

**Review by Willyfrog — APPROVED** [review]

**Comment by amyblais** [comment]
Devin, I assigned this to you because I don't have merge permissions here.

**Pull request merged**